src/App.tsx

```diff
@@ -26,70 +27,119 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
```

Comment:
To specify the type of function arguments, it is better not to divide types into native (```goods: string[]```) and separately created (```{ sortType, isReversed }: ReorderOptions```), but to write a single type or interface for function arguments as a whole:
```
function getReorderedGoods(options: ReorderOptions) { ... }
```